tiledb/sm/query/query_condition.h

```diff
@@ -58,11 +59,11 @@ class QueryCondition {
   struct Clause {
     /** Value constructor. */
     Clause(
-        std::string&& field_name,
+        std::string field_name,
```

Comment:
Why did this change from rvalue reference?  Maybe should be a const reference, and the `field_name_` initializer could remain the same.

Reply:
Resolved